### DISCUSSION TITLE
Turbo 'design-system' example w/ Storybook 7 + vanilla-extract ?

### DISCUSSION THREAD

**Discussion by jonnystening** [vercel/turborepo#4119] [Help] [answered]
Hey all

Wondering if anyone has managed to get the https://github.com/vercel/turbo/tree/main/examples/design-system example running with Storybook 7 and vanilla-extract?

Can do Turbo DS example with Storybook 7, but getting this error when hooking up vanilla-extract...

```
Styles were unable to be assigned to a file. This is generally caused by one of the following:

- You may have created styles outside of a '.css.ts' context
- You may have incorrect configuration. See https://vanilla-extract.style/documentation/getting-started
```

**Comment by jonnystening** [comment, marked as answer]
Hmm.... in my case it seems to be related to the way components are imported in story files. The import path needs to include the`/src/`.

e.g.
`import { Button } from "@acme/core/src";`

The dynamic alias I'd set up in the storybook vite config was missing the `/src` 🤦‍♂️

I have story files alongside component files, slightly different file structure to the Turbo 'design-system' example. My components are packaged individually and also prefer importing my components via package name (like in the real world).

Anyway, got it working like this...

packages/components/button/src/Button.stories.tsx
```
import { Button } from "@acme/button";
...
```

apps/docs/.storybook/main.ts
```
import { vanillaExtractPlugin } from '@vanilla-extract/vite-plugin';

...
async viteFinal(config) {
  return mergeConfig(config, {
    resolve: {
      alias: [
        {
          find: /^@acme\/(.*)/,
          replacement: '../../$1/src',
        },
      ],
    },
    plugins: [vanillaExtractPlugin()],
  });
},
...
```

Feel free to let me know if there's another/better way 👍

**Reply by AnweshGangula** [reply]
Seems like the [Original example](https://github.com/vercel/turborepo/blob/main/examples/design-system/apps/docs/.storybook/main.js) doesn't use vanilla-extract anymore
